var/spack/repos/builtin/packages/gdb/package.py

```diff
@@ -56,6 +56,7 @@ def install(self, spec, prefix):
         options = ['--prefix=%s' % prefix]
         if '+python' in spec:
             options.extend(['--with-python'])
+        options.append('LDFLAGS=-L{0}'.format(spec['python'].libs.ldflags))
```

Comment:
Sorry, it's actually `libs.ld_flags` with an underscore. Also, it automatically adds the `-L` for you, so you don't need that anymore. You should prob test it to make sure it actually works.

Reply:
build went through

Reply:
Builds with python3 but requires #7103 @adamjstewart you had a suggestion of how to fix `gdb`? It fails to pick up `python-config` because w/o #7103 it is only `python3-config`

Reply:
Does it have a flag you can use to specify the `python-config` file? If not, there's not much we can do aside from wait on #7103 and complain to the `gdb` devs.

Reply:
Not that I'm aware of... I wasn't able to find it immediately